src/components/ExpandableBox/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { CSSProperties, ReactNode, useRef, useState } from 'react'
+import styled from 'styled-components'
+
+import { ReactComponent as DropdownSVG } from 'assets/svg/down.svg'
+import { AutoColumn } from 'components/Column'
+import Divider from 'components/Divider'
+import { RowBetween } from 'components/Row'
+
+const Wrapper = styled(AutoColumn)`
+  & * {
+    transition: all ease-in-out 0.3s;
+  }
+`
+
+const Header = styled(RowBetween)<{ $expanded: boolean }>`
+  cursor: pointer;
+  z-index: 1;
+`
+
+const Content = styled.div<{ $expanded: boolean; $height: number }>`
+  padding: 10px 0;
+  ${({ $expanded, $height }) =>
+    $expanded
+      ? `margin-top:0; opacity:1; max-height:500px;`
+      : `margin-top:-${$height || 20}px; opacity:0; max-height:0;`}
+  z-index: 0;
+`
+export default function ExpandableBox({
```

Comment:
this component maybe seem like `components/collapse`

Reply:
I'm following this

Reply:
@XiaoYhun can you help check this